.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+## PR Checklist
```

Comment:
@lekhmanrus could you drop this PR template from this PR? For now I don't think we've been having problems with PRs not providing enough context, so I'd prefer to avoid the boilerplate.

Reply:
done